builder/solver.go

```diff
@@ -85,6 +87,11 @@ func (s *solver) newSolveOptMulti(ctx context.Context, eg *errgroup.Group, onIma
 		cacheExports = append(cacheExports, newInlineCacheOpt())
 	}
 
+	verboseProgressConsole := console.WithPrefixAndSalt("output",
+		"local context .", // TODO this salt must be the same as the salt used in SolverMonitor.processStatus
+	)
```

Comment:
BTW, I don't think this has to use the same salt - because the prefix is anyway different (context vs output). I would just use `"output", "output"` for now.

Reply:
When I did that I ended up with different coloured prefixes, which is what led me to use the `WithPrefixAndSalt` function.

Reply:
Different from what? Can you say more?

Reply:
e.g. one was orange, the other was blue.